services/version-api/helpers/decideVersionStatus.js

```diff
@@ -0,0 +1,41 @@
+import smCompare from 'semver-compare';
+
+import { VERSION_STATUS } from '../constants';
+
+const COMPARISON_RESULT = {
+  EQUAL: 0,
+  SMALLER: -1,
+  GREATER: 1,
+};
+
+/**
+ * Decides the status to return depending on currentVersion
+ *
+ * @param {string} currentVersion The current application version (in semver format)
+ * @param {string} minVersion The minimum allowed application version (in semver format)
+ * @param {string} maxVersion The latest application version released (in semver format)
+ * @returns {string} Version status
+ */
+function decideVersionStatus(currentVersion, minVersion, maxVersion) {
```

Comment:
We have previously agreed that comments on functions should not be written. The function names should have as explanatory name as possible.

It is good to try to keep the number of arguments to as few as possible as this facilitates testing of the function. Preferably 1 but 2 is ok